File: Java/Unique-Characters.java
public class Solution {
    /**
     * @param str: a string
     * @return: a boolean
     */
    public boolean isUnique(String str) {
        // write your code here
        int[] vis = new int[256];
        for (int i = 0; i < str.length(); i ++) {
            if(vis[str.charAt(i)] ++ > 0) return false;
        }
        return true;
    }
}

File: Cpp/Unique-Characters.cpp
#include <string>
#include <set>

class Solution {
public:
    /**
     * @param str: a string
     * @return: a boolean
     */
    bool isUnique(string &str) {
        int vis[256];
        memset(vis, 0, sizeof(vis));
        for (int i = 0; i < str.size(); i++){
            if(vis[str[i]] == 0) vis[str[i]] = 1;
            else return false;
        }
        return true;
        // write your code here
    }
};
